### DISCUSSION TITLE
WingPanel Customization

### DISCUSSION THREAD

**Discussion by arun0345** [elementary/wingpanel#474] [Q&A]
Hi ,
in odin release pantheon tweaks are having minimal features than its previous releases. not only wingpanel but lot of tools lacks customization. for ex (code editor, files, icon zoom in dock, etc.)  it would great to have included  customization in default apps.

Thanks!!

**Comment by davidmhewitt** [comment]
elementary does not maintain Pantheon tweaks.

**Reply by arun0345** [reply]
Thanks for ur reply. but still its good to have included some customization for wingpanel & docks in elementary
